src/server/config/env/runTime.js

```diff
@@ -199,6 +199,11 @@ const runTime = [
     normalize: (value) => value === 'true',
     defaultValue: () => false,
   },
+  {
+    name: 'ONE_FALLBACK_URL',
```

Comment:
could we please add some docs for this and how it can be used?

Reply:
@nellyk as mentioned in the description of the PR, I was waiting for the approvals so that I knew the implementation was 100% before writing the docs. I didn't want to write the docs and then have to re-write them because something changed due to reviews.